### ISSUE TITLE
Adjustable font-sizes for accessibility

### ISSUE THREAD

**Issue by CollinsSpencer** [UNL-CSE-AMBASSADORS/VITA#56]
The problem with pixels is that they don't scale based on browser settings. Some people who have poor eye-sight set their browsers to have a default font size that is larger, which doesn't change the scale for how large "pixels" are. That's the difference between the browser's zoom feature and the accessibility font-size feature. Everything scales with the zoom.

The problem I'm seeing is for mobile users where the zoom feature isn't an option. They can zoom in, but it's more like a magnifying glass. Some people don't know how to turn on the larger text accessibility feature (which is ironic). That makes me wonder if on mobile there should be an button that can make the text larger right in the page. Reasonably, that would only affect the font-size on the page, but maybe not the nav bar?
